runtime/runtime_test.go

```diff
@@ -0,0 +1,184 @@
+package runtime
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"syscall"
+	"testing"
+	"time"
+
+	utils "github.com/docker/containerd/testutils"
+)
+
+var (
+	referenceSpec = "config.json"
+	devNull       = "/dev/null"
+	stdin         io.WriteCloser
+)
+
+func generateReferenceSpecs(destination string) error {
```

Comment:
Done. There's more code that can be extracted out from integration-test setup code into package testutils. Lets do that in a follow up PR.